sonic_platform_base/sonic_sfp/sfputilhelper.py

```diff
@@ -46,6 +52,30 @@ def read_porttab_mappings(self, porttabfile, asic_inst=0):
         port_pos_in_file = 0
         parse_fmt_port_config_ini = False
 
+        if get_ports_from_db:
+            (platform, hwsku) = device_info.get_platform_and_hwsku()
+            (ports, _, _) = get_port_config(hwsku, platform, get_ports_from_db=True)
+            if not ports:
+                helper_logger.log_error("failed to get ports info from config DB")
+
+            for port_alias in ports.keys():
+                self.logical.append(port_alias)
+
+                if 'index' in ports[port_alias].keys():
+                    port_idx = int(ports[port_alias]['index'])
+                    self.logical_to_physical[port_alias] = [port_idx]
+
+                    if self.physical_to_logical.get(port_idx) is None:
+                        self.physical_to_logical[port_idx] = [port_alias]
+                    else:
+                        self.physical_to_logical[port_idx].append(port_alias)
+
+                    self.logical_to_asic[port_alias] = asic_inst
+                else:
+                    helper_logger.log_error("failed to parse ports info, port index is missing")
+
+            return None
```

Comment:
Will be better to use if else instead of returning None here.
single 'return' is also an option

Reply:
Done.